.github/workflows/actions-test.yml

```diff
@@ -32,4 +32,42 @@ jobs:
         if: steps.cache-node-modules.outputs.cache-hit != 'true'
         run: npm install
       - name: Run unit tests with coverage
-        run: npm run test
\ No newline at end of file
+        run: npm run test
+  e2e-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.12.x
+          cache: npm
+      - name: Start test containers
+        run: docker compose -f "./e2e/helpers/docker/docker-compose.yml" up -d
+      - name: Cache node modules
+        id: cache-node-modules
+        uses: actions/cache@v4
+        with:
+          path: node_modules
+          key: node-modules-${{ hashFiles('package-lock.json') }}
+      - name: Install Dependencies
+        if: steps.cache-node-modules.outputs.cache-hit != 'true'
+        run: npm install
+      - name: Start indexer
+        run: npm run start:e2e > indexer.log 2>&1 &
+      - name: Wait for indexer to become available
```

Comment:
Seems Good to me . What is your opinion regarding putting indexer also in a container ?

Reply:
Do you mean, creating a Dokcerfile, building the image, and running that image here?